package.json

```diff
@@ -7,6 +7,7 @@
     "test": "tests"
   },
   "scripts": {
+    "debug": "./node_modules/ts-node/dist/bin.js src/main.ts -t",
```

Comment:
Is this intentional, or leftover from your debugging?

Reply:
Intentional. I have a `local_test` directory on my local repo (hidden to git) and this makes it easy to make tests without touching the "real" tests. I would prefer to keep it (for my own convenience).

Reply:
I have renamed it to `test` though, to make it clearer.